### PR TITLE
PD-3514 fixing forwarded emails not creating PRM contact records

### PR DESCRIPTION
**Pull request by vigowei**
## What's this PR do?
de-duplicate contact emails in email processing

## Any background context you want to provide?
Remember to add `QC_EMAIL_KEY`

### CODE REVIEW

File: deploy/settings.myjobs_qc.py [hunk 1]
```diff
@@ -97,6 +97,8 @@
 EMAIL_HOST_USER = QC_EMAIL_HOST_USER
 EMAIL_HOST_PASSWORD = QC_EMAIL_HOST_PASSWORD
 
+EMAIL_KEY = QC_EMAIL_KEY
```
Comment (mklauber) [inline deploy/settings.myjobs_qc.py]: Where is this coming from?  Should be imported or defined somewhere?

Comment (vigowei) [inline deploy/settings.myjobs_qc.py]: I think this should be added to the secrets file, @almathew can confirm

Comment (vigowei) [inline deploy/settings.myjobs_qc.py]: I previously thought it would come from default settings but secrets makes more sense

Comment (almathew) [inline deploy/settings.myjobs_qc.py]: I added it to the secrets file yesterday. It is a secret.

**Pull request merged**